reactioncommerce/reaction
# Fix for error while archiving a product

This PR resolves [#2620](https://github.com/reactioncommerce/reaction/issues/2620). The said issue was a bug report for an error that occurred whenever a user tried to archive a product. These trials consequently failed. And it was all due to the wrong use of a certain function's parameter.

## Testing steps

1. Login as admin.
1. Click *once* on the default product (i.e `Basic Reaction Product`) or another product (if you've added any).
1. Select "Clone product" in the side menu.
1. *Double-click* the newly created product to open it (scroll the page to find it if necessary).
1. Click on "Archive" in the top menu.
1. The archiving operation should now be completed without any errors. Similarly, check the terminal/command prompt and the error reported in [#2620](https://github.com/reactioncommerce/reaction/issues/2620) should be absent.
